### ISSUE TITLE
[xwtorrents] Exception (xwtorrents): The cookies provided by FlareSolverr are not valid: The cookies provided by FlareSolverr are not valid (Test)

### ISSUE THREAD

**Issue by rgallo** [FlareSolverr/FlareSolverr#169]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

### How to enable debug and html traces

[Follow the instructions from this wiki page](https://github.com/FlareSolverr/FlareSolverr/wiki/How-to-enable-debug-and-html-trace)

### Environment

* **FlareSolverr version**: Latest (1.2.8)
* **Last working FlareSolverr version**: N/A
* **Operating system**: Linux (Synology)
* **Are you using Docker**: yes
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** no
* **If using captcha solver, which one:**
* **URL to test this issue:** https://xtremewrestlingtorrents.net

### Description

Error in title

### Logged Error Messages
```
> flaresolverr@1.2.8 start

> node ./dist/index.js


2021-07-13T20:47:43+00:00 INFO FlareSolverr v1.2.8

2021-07-13T20:47:43+00:00 DEBUG Debug log enabled

2021-07-13T20:47:43+00:00 DEBUG Testing Chrome installation...

2021-07-13T20:47:43+00:00 DEBUG Launching browser...

2021-07-13T20:47:49+00:00 DEBUG Test successful

2021-07-13T20:47:49+00:00 INFO Listening on http://0.0.0.0:8191

2021-07-13T20:47:59+00:00 INFO REQ-1 Incoming request: POST /v1

2021-07-13T20:47:59+00:00 INFO REQ-1 Params: {"maxTimeout":60000,"cmd":"request.get","url":"https://xtremewrestlingtorrents.net/browse.php?incldead=1&search=&sort=4&type=desc","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}

2021-07-13T20:47:59+00:00 DEBUG REQ-1 Launching browser...

2021-07-13T20:48:00+00:00 DEBUG REQ-1 Using custom UA: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36

2021-07-13T20:48:00+00:00 DEBUG REQ-1 Adding custom headers: {}

2021-07-13T20:48:00+00:00 DEBUG REQ-1 Navigating to... https://xtremewrestlingtorrents.net/browse.php?incldead=1&search=&sort=4&type=desc

2021-07-13T20:48:01+00:00 DEBUG REQ-1 Overrides: {"headers":{"upgrade-insecure-requests":"1","user-agent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36"}}

2021-07-13T20:48:04+00:00 DEBUG REQ-1 <html><head><link type="text/css" rel="stylesheet" href="clock_assets/flipclock.css">



<link rel="Shortcut Icon" href="/favicon.ico">

<script type="text/javascript" src="/js/balloontip.js"></script>

<script async="" src="/cdn-cgi/bm/cv/669835187/api.js"></script><link rel="Shortcut Icon" href="/favicon.ico"><script type="text/javascript" src="js/jquery.min.js"></script><script type="text/javascript" src="java_klappe.js"></script><title>

������
XWT :: Login

</title><link rel="stylesheet" href="/default-new.css" type="text/css"><link href="styles-neomedia.css" rel="stylesheet" type="text/css"><script type="text/javascript" language="javascript1.2" src="ajax.js"></script></head>










 




<!-- /*******************************     Cambiar direccion      ***********************************/  -->





<body><center><img src="/2021banners/rotate.php"></center>


<table class="mainouter" width="1286px" cellspacing="0" cellpadding="0">


      </table><table align="center" cellpadding="0" cellspacing="0">


          <tbody><tr><td rowspan="2"><a href="https://xtremewrestlingtorrents.net/index.php"><img src="/pic/home.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/browse.php"><img src="/pic/browse.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/upload.php"><img src="/pic/upload.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/my.php"><img src="/pic/profile.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/forums.php"><img src="/pic/forums.png"></a></td>		


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/static.php"><img src="/pic/stats.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/invite.php"><img src="/pic/invite.png"></a></td>		 


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/rules.php"><img src="/pic/rules.png"></a></td>


          <td rowspan="2"><a href="https://xtremewrestlingtorrents.net/donate.php"><img src="/pic/donate.png"></a></td>

		  

		  		  <td rowspan="2"><a href="chat.php"><img src="/pic/chat.png"></a></td>


          <td rowspan="2"><a href="staff.php"><img src="/pic/staff.png"></a></td>


          <td rowspan="2"><a href="https://xwt-classics.net"><img src="/pic/classics.png"></a></td>


        </tr>


      </tbody></table>     

  





<table width="1286px" class="mainouter" cellspacing="0" cellpadding="0" align="center">


<!------------- MENU ------------------------------------------------------------------------>



<tbody><tr>


  <td align="center"><div class="left" id="menu-up">


    </div></td>


</tr>



<tr><td align="center" class="outer" style="padding-top: 0px; padding-bottom: 20px">


<br>


<br><h1>Not logged in!</h1>

<p><b>Error:</b> The page you tried to view can only be used when you're logged in.</p>

<form method="post" action="takelogin.php">

<p>Note: You need cookies enabled to log in.</p>

<table border="0" cellpadding="5">

<tbody><tr><td class="rowhead">Username:</td><td align="left"><input type="text" size="40" name="username"></td></tr>

<tr><td class="rowhead">Password:</td><td align="left"><input type="password" size="40" name="password"></td></tr>

<!--<tr><td class=rowhead>Duration:</td><td align=left><input type=checkbox name=logout value='yes' checked>Log me out after 15 minutes inactivity</td></tr>-->

<tr><td colspan="2" align="center"><input type="submit" value="Log in!" class="btn"></td></tr>

</tbody></table>

<input type="hidden" name="returnto" value="/browse.php?incldead=1&amp;search=&amp;sort=4&amp;type=desc">

</form><br>

<h2><a href="signup.php">Registrations are Open!</a></h2><br>

<h2><a href="recover.php">Forgot your password? Click here</a></h2>






    <script type="text/javascript">(function(){window['__CF$cv$params']={r:'redacted',m:'redacted',s:[redacted],}})();</script>

</td></tr></tbody></table></body></html>

2021-07-13T20:48:04+00:00 INFO REQ-1 Cloudflare detected

2021-07-13T20:48:04+00:00 DEBUG REQ-1 Javascript challenge not detected. Status code: 200

2021-07-13T20:48:06+00:00 INFO REQ-1 Challenge solved.

2021-07-13T20:48:06+00:00 INFO REQ-1 Response in 7.26 s
```

**Comment by Ibbanez** [comment]
I have this as well, so I don't want to create a duplicate issue.

**Comment by fgordon-maker** [comment]
I'm getting this as well.

**Comment by dRuEFFECT** [comment]
+1
Are you guys getting a "Unsupported POST" error when trying to recreate XWTorrents config from scratch? that's what i've been running into and only getting the invalid cookies error when hitting the "Test" button on a previously saved XWT config.
Here's the ticket I opened on the Jackett repo...
https://github.com/Jackett/Jackett/issues/12051

**Comment by ngosang** [comment]
I'm not getting a Cloudflare challenge for this site so I can not reproduce the issue. The site is working fine without Cloudflare.

**Comment by dRuEFFECT** [comment]
> I'm not getting a Cloudflare challenge for this site so I can not reproduce the issue. The site is working fine without Cloudflare.

this is what I get when taking out the flaresolverr url.... from config... why would you not be getting a challenge?
![image](https://user-images.githubusercontent.com/7904175/128133297-08845033-8caf-458b-8dee-d46399612bc0.png)

**Comment by dRuEFFECT** [comment]
@ngosang can we remove the more information needed tag? i'm showing you proof that jackett is complaining of cloudflare challenge without flaresolverr.. if you need more information than this, i'd be happy to provide.

**Comment by ngosang** [comment]
Fixed in [FlareSolverr v2.0.0](https://github.com/FlareSolverr/FlareSolverr/releases/tag/v2.0.0)
If you are using Jackett, version v0.18.1041 or higher is required to bypass Cloudflare.
